src/locales/en-US/commands.json

```
@@ -1412,5 +1412,11 @@
   "crappydesign": {
     "commandDescription": "Shows a random image from r/CrappyDesign.",
     "title": "Here's your crappy design."
+  },
+  "tunein": {
+    "commandDescription": "Search for radio stations",
+    "commandUsage": "<query>",
+    "nowPlaying": "Now Playing:",
```

```suggestion
    "nowPlaying": "Now Playing",
```

Our other commands don't have `:` in field titles